apps/prairielearn/elements/pl-drawing/pl-drawing.py

```diff
@@ -35,13 +35,13 @@ def union_drawing_items(e1, e2):
     return newobj
 
 
-def load_extensions(data):
+def load_extensions(data: pl.QuestionData) -> None:
     extensions = pl.load_all_extensions(data)
     for name, ext in extensions.items():
         elements.register_extension(name, ext, data)
 
 
-def check_attributes_rec(element):
+def check_attributes_rec(element: lxml.html.HtmlElement):
```

Comment:
I think this returns none?

Reply:
Resolved in [09880cb](https://github.com/PrairieLearn/PrairieLearn/pull/11855/commits/09880cbfda46aab2fa9baf0356868b970cdde83d).